README.md

```diff
@@ -746,8 +746,8 @@ Having said this, graphql-http is mostly aimed for library authors and simple se
 
 If you want a feature-full server with bleeding edge technologies, you're recommended to use one of the following.
 
-<!-- <ServersTable> -->
 <!-- prettier-ignore-start -->
+<!-- <ServersTable> -->
 | Name | Audit |
```

Comment:
Why reorder? This table is autogenerated by the [render-servers-table.mjs script](https://github.com/graphql/graphql-http/blob/54fdb10/scripts/render-servers-table.mjs).

Reply:
Please do, the script will take care of it.